integration/key/key.go

```diff
@@ -0,0 +1,11 @@
+package key
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/aws-operator/integration/env"
+)
+
+func HostPeerStackName() string {
+	return fmt.Sprintf("host-peer-%s", env.ClusterID())
+}
```

Comment:
Having separate `key` package doesn't make sense to me practically. I understand logical reasons. But `env` isn't only about taking values from env variables. I would just put everything there.

Reply:
I was complaining there as well.